From fd86320b9cc03ae1616308bd26f973b7fe46c02a Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Fri, 31 Aug 2012 17:19:24 +0800
Subject: ENGR00222197 MX6 SabreSD:Set pwm backlight max density to 248

This patch changes pwm backlight max density from 255 to 248
to workaround Hannstar LVDS panel unstable backlight issue
when density is set to 250 or 251.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
(cherry picked from commit 8747626ca0bcdb6c9525e28d3fbb170db462a299)
---
 arch/arm/mach-mx6/board-mx6q_sabresd.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabresd.c b/arch/arm/mach-mx6/board-mx6q_sabresd.c
index 59ea80c..e198617 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabresd.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabresd.c
@@ -1575,7 +1575,7 @@ static void __init imx6q_add_device_buttons(void) {}
 
 static struct platform_pwm_backlight_data mx6_sabresd_pwm_backlight_data = {
 	.pwm_id = 0,
-	.max_brightness = 255,
+	.max_brightness = 248,
 	.dft_brightness = 128,
 	.pwm_period_ns = 50000,
 };
-- 
1.7.9.5

